generators/server/templates/src/main/docker/jhipster-control-center.yml.ejs

```diff
@@ -0,0 +1,58 @@
+<%#
+Copyright 2013-2020 the original author or authors from the JHipster project.
+
+This file is part of the JHipster project, see https://www.jhipster.tech/
+for more information.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-%>
+## How to use JHCC docker compose
+### Discovery mode
+# JHCC support 3 kinds of discovery mode: Consul, Eureka and static
+# In order to use one, please set SPRING_PROFILES_ACTIVE to one (and only one) of this values: consul,eureka,static
+### Discovery properties
+# According to the discovery mode choose as Spring profile, you have to set the right properties
+# please note that current properties are set to run JHCC with default values, personalize them if needed
+# and remove those from other modes. You can only have one mode active.
+#### Eureka
+#       - EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=http://admin:admin@host.docker.internal:8761/eureka/
```

Comment:
we can add this by default, when discoverytype is eureka, then, we can suppose the registry is started with docker, so the host should be jhipster-registry
we can anyway keep this line as commented, in case the registry is started direct from host, instead of docker

Reply:
I wonder if we should keep host.docker.internal as it should work for every cases (to be confirm)

Reply:
I think it's best to move this comment to the docs and generate only the relevant option (static/eureka/consul). This means we need to write a page on the JHipster Control Center !

Reply:
For sure, we need a dedicated page for JHCC. Now the yaml is adjusted according to the project configuration we can move out this or add only what we need.

Reply:
https://github.com/jhipster/generator-jhipster/issues/12163